lib/twitter/rest/request.rb

```diff
@@ -26,6 +27,7 @@ def initialize(client, request_method, path, options = {})
         @client = client
         @uri = Addressable::URI.parse(path.start_with?('http') ? path : BASE_URL + path)
         set_multipart_options!(request_method, options)
+        timeout_options!(options)
```

Comment:
I originally wrote it as `set_timeout_options!` and ran `bundle exec rake`.  That failed because they're using Rubocop to validate the code, and Rubocop doesn't like `set_`.  Rubocop did NOT report a problem with `set_multipart_options!`, but I don't understand why.

I'm not SURE that the official build will fail if Rubocop reports errors- I didn't look into it.  But the authors seemed to want Rubocop to validate, so that's what I did.  They may well prefer consistency over Rubocop validity, I'm not sure.